.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
+
+on:
+  [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
`test` might be a more accurate name for this job. Or you could add a `name: ...` mapping to have a longer, human-readable name for the job.